include/Util/BasicTypes.h

```diff
@@ -130,19 +130,21 @@ class SVFFunction : public SVFValue
     bool isIntri;
     Function* fun;
     std::vector<const BasicBlock*> reachableBBs;
+    mutable bool isUncalled;
+    mutable bool isNotRet;
```

Comment:
I know that `setIsUncalled` is called below, but is this necessary, as in, is there a way to achieve the same thing without using `mutable`? Can we know these values at construction time. `mutable` is a bit scary haha.